pages/api/indexer/id_to_infts.ts

```diff
@@ -0,0 +1,36 @@
+import type { NextApiRequest, NextApiResponse } from "next";
+import { connectToDatabase } from "../../../lib/mongodb";
+import NextCors from "nextjs-cors";
+import { QueryError } from "../../../types/backTypes";
+
+type IdentityNFT = {
+  contract: string,
+  inft_id: string,
+};
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse<Array<IdentityNFT> | QueryError>
+) {
+  await NextCors(req, res, {
+    methods: ["GET"],
+    origin: "*",
+    optionsSuccessStatus: 200,
+  });
+  const {
+    query: { id },
+  } = req;
+  const { db } = await connectToDatabase();
+  const domainCollection = db.collection("equipped_infts");
+  const output: Array<IdentityNFT> = [];
+  await domainCollection
+    .find({ starknet_id: id, "_chain.valid_to": null })
+    .forEach((doc) => {
+      output.push({ contract: doc.contract, inft_id: doc.inft_id });
```

Comment:
Don't you think it's better to add also a ``description`` + a ``get_uri`` ?

Reply:
So what you propose is that we simply get the Id and the on the UI we do all the work to get the data ?

Reply:
We call the get_uri function which will return us the json containing image and description